ci-operator/jobs/openshift-priv/cluster-bootstrap/openshift-priv-cluster-bootstrap-main-presubmits.yaml

```diff
@@ -96,7 +96,7 @@ presubmits:
         name: github-credentials-openshift-ci-robot-private-git-cloner
     hidden: true
     labels:
-      ci-operator.openshift.io/cloud: equinix-edge-enablemnt
+      ci-operator.openshift.io/cloud: equinix-edge-enablement
```

Comment:
Better to fix typo in another PR. There will be interference with this PR and the related merging will be delayed

Reply:
> Better to fix typo in another PR. There will be interference with this PR and the related merging will be delayed

That was not mine when I does make update these changes came. I will do rebase when current jobs completed.